lib/elixir/lib/kernel.ex

```diff
@@ -6212,7 +6212,7 @@ defmodule Kernel do
   """
   @doc since: "1.17.0"
   @spec to_timeout([component, ...] | timeout() | Duration.t()) :: timeout()
-        when component: [{unit, non_neg_integer()}, ...],
```

Comment:
I wonder if we can just remove `component` and inline the tuple, might be easier to read actually?